src/main/java/org/isf/help/HelpViewer.java

```diff
@@ -40,28 +43,36 @@ public class HelpViewer extends JDialog {
 	private static final long serialVersionUID = 1L;
 
 	private static final String MANUAL_PDF_FILE = "UserManual.pdf";
+	private static final String USER_MANUAL_URI = "https://www.open-hospital.org/user-manual";
```

Comment:
I think it could be a good idea to link to the corresponding version on GitHub, for instance:

https://github.com/informatici/openhospital-doc/blob/1.11.5/doc_user/UserManual.adoc

taking the `1.11.5` from `version.properties`

Reply:
I disagree with that approach; it looks less official and I'm afraid people will be confused by the GIithub banners and other info there:
![image](https://user-images.githubusercontent.com/1105445/213882506-2aaab9e2-6f16-4f07-987e-83258ef1ea66.png)

Reply:
I would assume must hospital staff don't even now what Github.   @mizzioisf Do you have an opinion?

Reply:
I don't think users would be confused by GitHub because they would go straight to the content.
The point is not GitHub or official website (GitHub is official too btw) but to link to the correct version of the doc.

We can think to publish each version in HTML somewhere online, I do remember that asciidoc was chosen to be ready for [readthedocs](https://readthedocs.org/) but I was trying without success (it needs an index.rst in the project)

Reply:
I used the link you provided to get the screen shot that I added.   Note the header, banners, icons, etc.   It just doesn't render the content.  It renders the content within the GitHub application.

Reply:
Yes, I know it very well, but it is the only solution we have so far for versioned doc

Reply:
It shouldn't be that hard to provide versioned documentation on the website; I've done it before.   But in the meantime I've changed it to go to Github.

Reply:
Yes as a temporary solution is better than nothing, thanks!